### PR TITLE
merge task 2

### PR DESCRIPTION
**Pull request by yairrim**

### CODE REVIEW

File: PushbotTeleopTank_Linear.java [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+    double          clawOffset  = 0.0 ;                  // Servo mid position
+    final double    CLAW_SPEED  = 0.02 ;                 // sets rate to move servo
+
+    /*
+     * Code to run ONCE when the driver hits INIT
+     */
+
+    public void runOpMode() {
+        while (opModeIsActive()) {
+            robot.init(hardwareMap);
```
Comment (benami-yassour) [inline PushbotTeleopTank_Linear.java]: The robot init should be before the while loop.

File: PushbotTeleopTank_Linear.java [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+            waitForStart();
```
Comment (benami-yassour) [inline PushbotTeleopTank_Linear.java]: The waitforstrart whould be before the while loop.

**Review by benami-yassour — COMMENTED** [review]
See some comments.

File: PushbotTeleopTank_Linear.java (hunk 1 above)
Comment (benami-yassour) [reply PushbotTeleopTank_Linear.java]: ping

Comment (yairrim) [reply PushbotTeleopTank_Linear.java]: done!

File: PushbotTeleopTank_Linear.java (hunk 2 above)
Comment (yairrim) [reply PushbotTeleopTank_Linear.java]: done!

File: PushbotTeleopTank_Linear.java [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+    final double    CLAW_SPEED  = 0.02 ;                 // sets rate to move servo
+
+    /*
+     * Code to run ONCE when the driver hits INIT
+     */
+
+    public void runOpMode() {
+        robot.init(hardwareMap);
+
+
```
Comment (benami-yassour) [inline PushbotTeleopTank_Linear.java]: At most single empty line.

File: PushbotTeleopTank_Linear.java [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+    /*
+     * Code to run ONCE when the driver hits INIT
+     */
+
+    public void runOpMode() {
+        robot.init(hardwareMap);
+
+        
+        waitForStart();
+
```
Comment (benami-yassour) [inline PushbotTeleopTank_Linear.java]: remove empty line

**Review by benami-yassour — CHANGES_REQUESTED** [review]

**Pull request merged**